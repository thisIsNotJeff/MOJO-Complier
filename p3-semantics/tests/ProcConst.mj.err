 line 4, column 9: error: nested procedures are not constants
 line 4, column 9: error: cannot assign nested procedures
