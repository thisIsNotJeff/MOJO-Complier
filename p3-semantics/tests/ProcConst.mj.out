BEGIN _BUILTIN
_BUILTIN.int=int
_BUILTIN.bool=bool
_BUILTIN.false:bool=false
_BUILTIN.true:bool=true
_BUILTIN.Null=^null
_BUILTIN.nil:Null=0
_BUILTIN.Refany=^null
_BUILTIN.Address=^null
_BUILTIN.Root=null
object {
}
_BUILTIN.Text=^null
_BUILTIN.First:(...)
_BUILTIN.Last:(...)
_BUILTIN.New:(...)
_BUILTIN.Number:(...)
BEGIN ProcConst
ProcConst.foo:()=ProcConst.Foo:()
ProcConst.Foo:()
BEGIN ProcConst.Foo
END ProcConst.Foo
BEGIN ProcConst.3
ProcConst.3.bar:()=ProcConst.3.Bar:()
ProcConst.3.Bar:()
BEGIN ProcConst.3.Bar
END ProcConst.3.Bar
END ProcConst.3
END ProcConst
END _BUILTIN
